src/filesystem/impls/makedrive/SampleFileSystem.js

```diff
@@ -1,5 +1,4 @@
 /*jslint vars: true, plusplus: true, devel: true, nomen: true, regexp: true, indent: 4, maxerr: 50 */
-/*global define, appshell, $, window */
```

Comment:
From what I can tell, these are global declarations to JSHint to tell it that those Globals are going to be used within the confines of that file, but in SampleFileSystem.js none of these Globals are being used.